themes/theme-ios11/components/Headline/index.jsx

```diff
@@ -1,18 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Ellipsis from '@shopgate/pwa-common/components/Ellipsis';
+import I18n from '@shopgate/pwa-common/components/I18n';
 import styles from './style';
 
 /**
  * The headline component.
- * @param {string} props The component props.
+ * @param {Object} props The component props.
  * @returns {JSX}
  */
-const Headline = ({ text }) => (
-  text.length ? <h3 className={styles.headline}>{text}</h3> : null
-);
+const Headline = (props) => {
```

Comment:
This component receives a bunch of props. Even though the `children` prop is hard to check, I would use `PureComponent` here.